test/e2e/operator_test.go

```diff
@@ -283,6 +283,21 @@ func TestClusterIngressUpdate(t *testing.T) {
 		t.Fatalf("failed to get default router deployment: %v", err)
 	}
 
+	// Wait for the CA certificate configmap to exist.
+	configmap := &corev1.ConfigMap{}
+	err = wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
+		if err := cl.Get(context.TODO(), types.NamespacedName{Namespace: ns, Name: "router-ca"}, configmap); err != nil {
```

Comment:
Shouldn't this be `globalMachineSpecifiedConfigNamespace`?

Reply:
Oops! You're right.  Fixed.